jupyter_server_terminals/app.py

```diff
@@ -73,9 +74,16 @@ def initialize_configurables(self) -> None:
 
     def initialize_handlers(self) -> None:
         """Initialize handlers."""
-        if not self.serverapp or not self.serverapp.terminals_enabled:
+        if not self.serverapp:
             # Checking self.terminals_available instead breaks enabling terminals
```

Comment:
Something should be done to this comment I previously added. Perhaps it should be moved or deleted. With the if statement split up, my comment no longer makes sense in this block, but might make sense in the next block

Reply:
Thanks!